src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -393,6 +393,99 @@ def test_azconfig_import_export(self, resource_group, location):
         assert imported_kvs == exported_kvs
 
 
+class AppConfigAppServiceImportExportLiveScenarioTest(LiveScenarioTest):
```

Comment:
Because you mark this test class as LiveScenarioTest, is it expected to you? May I know the reason that you want to make it live?

Reply:
These tests only pass when we run them live. We are fine with making this LiveScenarioTest because we don't expect major code changes in this area.